Tests/UnitTests/Core/Geometry/ProtoLayerTests.cpp

```diff
@@ -138,20 +138,18 @@ BOOST_AUTO_TEST_CASE(ProtoLayerTests) {
 
   std::stringstream sstream;
   protoLayerRot.toStream(sstream);
-
   std::string oString = R"(ProtoLayer with dimensions (min/max)
 Extent in space : 
   - value :      binX | range = [-6.66104, 6.66104]
   - value :      binY | range = [-4.85241, 4.85241]
   - value :      binZ | range = [-6, 6]
   - value :      binR | range = [3, 6.7082]
   - value :    binPhi | range = [-3.02295, 2.33295]
-  - value :   binRPhi | range = [0, 0]
+  - value :   binRPhi | range = [-20.2785, 15.6499]
```

Comment:
Did that change with one of the changes from the PR? Or did the test not work correctly before?

Reply:
This changed due to adding the missing enum in one of the switch statements.